### PR TITLE
fix h2 integration test

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

turbo-frame changes broke the h2 integration test;  this fixes it.

### CODE REVIEW

**Review by jmartin-sul — APPROVED** [review]
👍 pending rubocop fix (feel free to merge when green)

File: spec/features/create_object_h2_spec.rb [hunk 1]
```diff
@@ -35,8 +35,14 @@
     # signal that the background processes have finished before trying to
     # deposit an item to a collection.
     reload_page_until_timeout!(text: "Edit #{collection_title}", as_link: true)
+
     # Deposit an item to the collection
-    find(:table, collection_title).sibling('button').click
+    coll_link = find_link(collection_title, match: :first)
```
Comment (jcoyne) [inline spec/features/create_object_h2_spec.rb:40]: Is it possible to do `find(:table, collection_title).sibling('turbo-frame').child('button').click`?

Comment (ndushay) [reply spec/features/create_object_h2_spec.rb:40]: I'll try it, but `.first` hasn't worked in some other contexts.

Comment (ndushay) [reply spec/features/create_object_h2_spec.rb:40]: it liked it!   I think John and I got all bolluxed up by turbo-frame itself not being a sibling.  Thanks.

**Review by jcoyne — APPROVED** [review]

**Review by jmartin-sul — APPROVED** [review]
approved again 🙂

**Pull request merged**